### PR TITLE
Update forms.mdx. Use `id` instead of `name` for `htmlFor`

### PR DESCRIPTION
**Pull request by yuyokk**
Team,

I've updated this page to point `Label`.`htmlFor` to [element id](https://html.spec.whatwg.org/multipage/forms.html#attr-label-for) instead of name.

Please let me know if it makes sense for you.
Thanks

### CODE REVIEW

File: packages/docs/src/pages/components/forms.mdx [hunk 1]
```diff
@@ -26,13 +26,13 @@ import {
   onSubmit={e => e.preventDefault()}>
   <Label htmlFor='username'>Username</Label>
   <Input
-    name='username'
+    id='username'
```
Comment (jxnblk) [inline packages/docs/src/pages/components/forms.mdx:29]: Thanks! These elements should probably have both `name` and `id` attributes, shouldn't they?

Comment (yuyokk) [reply packages/docs/src/pages/components/forms.mdx:29]: As far as I know `name` can be skipped for labels access. If you want, I can add it back :)

Comment (yuyokk) [reply packages/docs/src/pages/components/forms.mdx:30]: @jxnblk added back. Let me know if anything else you think we need to change.

**Comment by jxnblk** [comment]
Thanks!

**Pull request merged**